sklearn/metrics/pairwise.py

```diff
@@ -1216,11 +1391,36 @@ def pairwise_distances(X, Y=None, metric="euclidean", n_jobs=1, **kwds):
                          "Valid metrics are %s, or 'precomputed', or a "
                          "callable" % (metric, _VALID_METRICS))
 
+    # To handle kill_missing = False
+    kill_missing = kwds.get("kill_missing")
```

Comment:
I don't get what this is all about. Why aren't we just adding 'masked_euclidean' to `PAIRWISE_DISTANCE_FUNCTIONS`?